cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskTracker.java

```diff
@@ -121,6 +135,7 @@ public void markTaskState(ExecutionTask task, ExecutionTaskState newState) {
         updateDataMovement(task);
         break;
       case COMPLETED:
+        _taskExecutionRateMeter.mark();
```

Comment:
I'm unclear if `executionRateMeter` will mark  completion regardless of the type of the task is being completed. In other words, leadership movements and partition movements captured by the same meter here? If yes, how can we distinguish?

Reply:
Yes it's not distinguish the task type. I think it's fine to have it for all task types.

Reply:
Can you please explain why distinguishing them is not necessary? If I recall correctly, Cruise-Control proposals contain a mix of partition movements, leadership movements, etc. The speed of execution of these different tasks is vastly different. 

For now, we could have just one cumulative meter. However, I think it would be useful to distinguish between the two so that they can be optimized separately, if needed.

Reply:
I believe all task types in a "rebalance" operation is INTER_BROKER_REPLICA_ACTION, including tasks like [a, b, c] -> [b, c, a].